records_mover/db/driver.py

```diff
@@ -56,7 +56,7 @@ def unload(self,
                schema: str,
                table: str,
                unload_plan: RecordsUnloadPlan,
-               directory: RecordsDirectory) -> int:
+               directory: RecordsDirectory) -> Optional[int]:
```

Comment:
Postgres doesn't supply the number of rows exported via its COPY TO statement.  Boo.